src/pages/BonusQuestion.tsx

```diff
@@ -21,7 +25,11 @@ const GET_BONUS_SCORE = gql`
 const BonusQuestion = () => {
   const history = useHistory()
 
-  const { setPointDifference, pointDifference } = useStore()
+  const { setPointDifference, pointDifference, addPoints } = useStore()
+
+  useIonViewWillEnter(() => {
+    addPoints(pointDifference)
```

Comment:
points are added upon entering the BonusQuestion screen, this should solve the animation issue